robottelo/ui/computeprofile.py

```diff
@@ -14,7 +14,13 @@ def search(self, name):
         return self.search_entity(name, locators['profile.select_name'])
 
     def select_resource(self, profile_name, res_name, res_type):
-        """Select necessary compute resource from specific compute profile"""
+        """Select necessary compute resource from specific compute profile
+        :param profile_name: Name of profile that contains required compute
+        resource (e.g. '2-Medium' or '1-Small')
+        :param res_name: Name of compute resource to select from the list
+        :param res_type: Type of compute resource (e.g. 'Libvirt' or 'Docker')
```

Comment:
Easy fix.

    """Select necessary compute resource from specific compute profile

    :param profile_name: Name of profile that contains required compute
        resource (e.g. '2-Medium' or '1-Small')
    :param res_name: Name of compute resource to select from the list
    :param res_type: Type of compute resource (e.g. 'Libvirt' or 'Docker')

    """